pkgs/applications/networking/cluster/kpt/default.nix

```
@@ -0,0 +1,29 @@
+{ buildGoModule, fetchFromGitHub,  fetchgit, fetchhg, fetchbzr, fetchsvn, lib }:
+
+buildGoModule rec {
+  pname = "kpt";
+  version = "0.31.0";
+
+  src = fetchFromGitHub {
+    owner = "GoogleContainerTools";
+    repo = "kpt";
+    rev = "v${version}";
+    sha256 = "1l5mpml6pf37b76wdq6il00k5q6rvw9ds7807m103k27p7pcqgdx";
+  };
+
+  modSha256 = "0y9s06qwh3g6a0x0mjxz5albkrnv1bvp3qrqw95hvmkrhh1073hc";
+
+  postFixup = ''
+    rm $out/bin/{mdtogo,formula}
+  '';
+
+  meta = with lib; {
+    description = "kpt is a toolkit to help you manage, manipulate, customize, and apply Kubernetes Resource configuration data files.";
```

This is a bit long. Also, see the contribution guidelines. The description should not start with the name of the package nor end with a period.

```suggestion
    description = "A toolkit to help you manage and apply Kubernetes Resource configuration data files";
```

---

Well see.